gcp/modules/backup-exporter/main.tf

```diff
@@ -0,0 +1,52 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "secrets_dir" {}
+variable "charts_dir" {}
+variable "destination_bucket" {}
+variable "project_id" {}
+variable "replica_count" {}
+variable "schedule" {}
+
+# Terragrunt variables
+
+data "google_project" "project" {
+  project_id = "${var.project_id}"
+}
+
+data "template_file" "backup-exporter" {
+  template = "${file("values.yaml")}"
+
+  vars {
+    service_account_name = "${data.google_service_account.backup_exporter.name}"
+    destination_bucket   = "${var.destination_bucket}"
+    replica_count        = "${var.replica_count}"
+    schedule             = "${var.schedule}"
+  }
+}
+
+module "backup-exporter" {
+  source           = "/exekube-modules/helm-release"
+  tiller_namespace = "kube-system"
+  client_auth      = "${var.secrets_dir}/kube-system/helm-tls"
+
+  release_name            = "backup-exporter"
+  release_namespace       = "backup-exporter"
+  release_values          = ""
+  release_values_rendered = "${data.template_file.backup-exporter.rendered}"
+
+  chart_name = "${var.charts_dir}/backup-exporter"
+}
+
+resource "google_storage_bucket" "backup_daisy_bkt" {
+  project = "${data.google_project.project.project_id}"
+  name    = "${data.google_project.project.name}-daisy-bkt"
```

Comment:
Daisy bucket?

Reply:
It's created by the automated cloudbuild task, it is not something that is easy to change.

Reply:
> created by the automated cloudbuild task

I don't see any cloud build resources in this PR – am I missing something?

Reply:
the command `gcloud compute image export` uses [Daisy](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy) which is a wrapper to the cloudbuild process, the command itself manages all the cloud build resources.